viewer/static/js/app/components/containers/dataset-control-container.js

```diff
@@ -65,7 +65,7 @@ class DatasetControlContainer extends React.Component {
         handleCohortSwitch={this._handleCohortSwitch}
         isBtnDisabled={this.isBtnDisabled}
 
-        currentDatasetId={this.props.selectedDatasetId}
+        currentDataset={this.props.selectedDataset}
```

Comment:
This is where the issue stems from. This prop has the same name as the prop on line 77, but they are two different types of data. This is a name, the one on line 77 is an object containing all sorts of data about the dataset. *dataset-control.js* needs the latter, so I'm going to push a commit that renames this.